contracts/MyMultiSig.cdc

```diff
@@ -353,6 +356,11 @@ pub contract MyMultiSig {
                     "Number of signers must be equal or higher than the threshold."
             }
 
+            for signer in _initialSigners {
+                // Checks if the signer exists, get account key will fail otherwise
+                getAccount(signer).keys.get(keyIndex: 0)
```

Comment:
same here, make it an `assert`.

no `pre` since looping through signers in a `pre` condition would be ugly/maybe not possible